src/pages/Programs.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Could you put this file in `components/program`? This is just a component as well

Reply:
i've moved it